exchanges/bitmex/bitmex_wrapper.go

```diff
@@ -348,30 +349,32 @@ func (b *Bitmex) UpdateOrderbook(p currency.Pair, assetType asset.Item) (*orderb
 		return nil, err
 	}
 
-	orderBook := new(orderbook.Base)
+	book := new(orderbook.Base)
 	for i := range orderbookNew {
-		if strings.EqualFold(orderbookNew[i].Side, order.Sell.String()) {
-			orderBook.Asks = append(orderBook.Asks, orderbook.Item{
+		switch {
+		case strings.EqualFold(orderbookNew[i].Side, order.Sell.String()):
+			book.Asks = append(book.Asks, orderbook.Item{
 				Amount: float64(orderbookNew[i].Size),
 				Price:  orderbookNew[i].Price})
-			continue
-		}
-		if strings.EqualFold(orderbookNew[i].Side, order.Buy.String()) {
-			orderBook.Bids = append(orderBook.Bids, orderbook.Item{
+		case strings.EqualFold(orderbookNew[i].Side, order.Buy.String()):
+			book.Bids = append(book.Bids, orderbook.Item{
 				Amount: float64(orderbookNew[i].Size),
 				Price:  orderbookNew[i].Price})
+		default:
+			return nil,
+				fmt.Errorf("could update orderbook order side could not be matched for %s",
```

Comment:
I'm not understanding what this error is saying. Could you help me out/update it?

Reply:
🤷 Will fix 😆